src/pythermalcomfort/models.py

```diff
@@ -1105,35 +1105,13 @@ def exponential(t_db):
 
     utci_approx = np.where(all_valid, utci_approx, np.nan)
 
-    return np.round_(utci_approx, 1)
-    #if utci_approx < -40:
-    #    stress_category = "extreme cold stress"
-    #elif utci_approx < -27:
-    #    stress_category = "very strong cold stress"
-    #elif utci_approx < -13:
-    #    stress_category = "strong cold stress"
-    #elif utci_approx < 0:
-    #    stress_category = "moderate cold stress"
-    #elif utci_approx < 9:
-    #    stress_category = "slight cold stress"
-    #elif utci_approx < 26:
-    #    stress_category = "no thermal stress"
-    #elif utci_approx < 32:
-    #    stress_category = "moderate heat stress"
-    #elif utci_approx < 38:
-    #    stress_category = "strong heat stress"
-    #elif utci_approx < 46:
-    #    stress_category = "very strong heat stress"
-    #else:
-    #    stress_category = "extreme heat stress"
-
-    #if units.lower() == "ip":
-    #    utci_approx = units_converter(tmp=utci_approx, from_units="si")[0]
-
-    #if return_stress_category:
-    #    return {"utci": round(utci_approx, 1), "stress_category": stress_category}
-    #else:
-    #    return round(utci_approx, 1)
+    if units.lower() == "ip":
```

Comment:
Should not we allow this function to get a numpy array as an input?

Reply:
`units_converter` already does accect numpy arrays, since it's just basic math snytax supported by numpy!